protobuf/protobuf.go

```diff
@@ -0,0 +1,150 @@
+package protobuf
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// Package represents an unique .proto file with its own package definition.
+type Package struct {
+	Name     string
+	Path     string
+	Imports  []string
```

Comment:
Now I understand that they are strings for the external package. Can we instead create an "external package" representation? This request can be disregarded and be followed up in an issue, though.